components/camel-graphql/src/main/java/org/apache/camel/component/graphql/Resources.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.camel.component.graphql;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.camel.RuntimeCamelException;
+
+public class Resources {
```

Comment:
I think we already a class in camel-support for reading file and stuff like that.

Reply:
OK, found a way to do it.